picard/tagger.py

```diff
@@ -412,7 +412,11 @@ def process(result=None, error=None):
                     self.add_files(result)
                 thread.run_task(get_files, process)
 
-        process(True, False)
+        if config.setting['recursively_add_files']:
+            process(True, False)
+        else:
+            self.add_files(os.path.join(path, f) for f in os.listdir(path))
```

Comment:
Trying this I saw the following error in the log:

> E: 17:55:57 Error occured:

I could only find "Error occured" in `picard/formats/__init__.py`. I guess this is because this also passes directory names to `add_files`.